assets/src/edit-story/components/carousel/carouselLayout.js

```diff
@@ -27,19 +27,21 @@ import { CarouselScrollForward, CarouselScrollBack } from './carouselScroll';
 import CarouselMenu from './carouselMenu';
 import CarouselList from './carouselList';
 import useCarousel from './useCarousel';
-import { MENU_GUTTER, BUTTON_GUTTER } from './constants';
+import { MENU_GUTTER, BUTTON_WIDTH, BUTTON_GAP } from './constants';
```

Comment:
Here, too -- there are two different general button sizes (not only related to the carousel) that seem to be needed elsewhere, too, so this should probably come from the design system (`buttons/index.js` or separate constants file) instead.

Reply:
Referring to the [previous discussion](https://github.com/google/web-stories-wp/pull/6272#discussion_r573286458).